.github/workflows/deploy.yml

```diff
@@ -40,26 +40,26 @@ jobs:
       - name: Push Docker image
         run: "docker push $(cat .docker_image):$(cat .docker_tag)"
       - name: deploy to dev-fss
-        uses: navikt/deployment-cli/action@a260e84
+        uses: navikt/deployment-cli/action@0.4.1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           cluster: dev-fss
           team: teamsykefravr
-          resource: naiserator-dev.yaml
+          resources: naiserator-dev.yaml
       - name: deploy to prod-fss
-        uses: navikt/deployment-cli/action@a260e84
+        uses: navikt/deployment-cli/action@0.4.1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           cluster: prod-fss
           team: teamsykefravr
-          resource: naiserator-prod.yaml
-      #- name: deploy redis to prod-fss
-      #  uses: navikt/deployment-cli/action@a260e84
-      #  env:
-      #    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      #  with:
-      #    cluster: prod-fss
-      #    team: teamsykefravr
-      #    resource: redis.yaml
\ No newline at end of file
+          resources: naiserator-prod.yaml
```

Comment:
Det at denne deployes sammen med appen er kanskje bare midlertidig nå for å få den ut? Eller tenkte du at den alltid skal deployes?

Reply:
Tenker at det er bare er midlertidig for å få den ut nå